CONTRIBUTORS.md

```diff
@@ -182,9 +182,8 @@ Symfony is the result of the work of many people who made the code better
  - Larry Garfield (crell)
  - Julien Falque (julienfalque)
  - Martin Schuhfuß (usefulthink)
- - apetitpa
+ - Arnaud Petitpas (apetitpa)
```

Comment:
You can revert this change because that file is autogenerated, so any change will be lost.